src/Image.cc

```diff
@@ -880,7 +880,11 @@ cairo_status_t
 Image::loadJPEG(FILE *stream) {
   cairo_status_t status;
 
+#if defined(_MSC_VER)
+  if (false) { // Force using loadJPEGFromBuffer
+#else
   if (data_mode == DATA_IMAGE) { // Can lazily read in the JPEG.
```

Comment:
Why not put the `#if defined(_MSC_VER)` inside of this this if-statment and avoid `if (false) {` and duplicating `if (data_mode == DATA_IMAGE) {`?